doc/source/user_guide/sparse.rst

```diff
@@ -6,27 +6,28 @@
 Sparse data structures
 **********************
 
-We have implemented "sparse" versions of ``Series`` and ``DataFrame``. These are not sparse
```

Comment:
I don't think the diff here is all that informative. I'd recommend just viewing the new file. The basic flow is

- short intro
- SparseArray / SparseDtype
- Sparse Accessors
- SparseIndex / computation
- Migration Guide
- SparseSeries / SparseDataFrame.